Source/WebCore/svg/SVGMarkerElement.idl

```diff
@@ -48,6 +49,7 @@
 
     undefined setOrientToAuto();
     undefined setOrientToAngle(SVGAngle angle);
+    undefined setOrientToAutoStartReverse();
```

Comment:
I do not see this method in the spec https://svgwg.org/svg2-draft/painting.html#InterfaceSVGMarkerElement. But it looks like adding the new enum value and adding this method are the only way to get this interface clean and complete. Please create a GitHub issue if there no one exists.